frontend/src/components/common/TooltipIcon.scss

```diff
@@ -4,6 +4,7 @@
   min-width: 14px;
   min-height: 14px;
   color: $secondary-variant-color;
+  margin-right: 1px;
```

Comment:
I just had a look in dev and noticed that all of the fields with tooltips don't have their inputs aligned. you can correct this by adding a negative margin equal to the width of the tooltip icon (you'll need to take into account the margin you've added above as well.